spec/ruby/core/module/refine_spec.rb

```diff
@@ -756,6 +756,32 @@ def foo
       result.should == [:A, :C]
     end
 
+    it "looks in the refined ancestors from included module" do
+      refined_class = ModuleSpecs.build_refined_class(for_super: true)
+      parent_class = Class.new(refined_class)
+
+      a = Module.new do
+        def foo
+          [:A] + super
+        end
+      end
+
+      refinement = Module.new do
+        refine refined_class do
+          include a
+        end
+      end
+
+      result = nil
+      Module.new do
+        using refinement
+
+        result = parent_class.new.foo
+      end
+
+      result.should == [:A, :C]
```

Comment:
Almost yes - without the PR we get an error (`foo': super: no superclass method foo' (NoMethodError)`), because there are no refinements in the `lookupSuperMethod` and we could not find a declartingModule

Reply:
Ah so it does find the refined method, but then doesn't find the super method of that refined method?

Reply:
Exactly :+1: 
The error is essentially the same as we fixed in the previous PR. The only difference is before all tests\discussions were about `C` and `C` refinements, and here we are talking about a subclass for C and `C` refinements,.